app/models/manageiq/providers/kubernetes/container_manager/refresher_mixin.rb

```diff
@@ -21,6 +21,18 @@ def fetch_entities(client, entities)
             end
           end
         end
+
+        def fetch_ems_metrics(ems)
+          hawk = ManageIQ::Providers::Kubernetes::ContainerManager::MetricsCapture::HawkularClient.new(ems)
+          keys = hawk.strings.query(:miq_metric => true)
+
+          metrics = {}
+          keys.each do |k|
```

Comment:
maybe you can use `each_with_objecct` here? instead of defining the metrcis hash and then returning it:
```
keys.each_with_object({|) do |k, metrics|
  .....
end
```